composer.json

```diff
@@ -50,7 +50,7 @@
         "bagisto/laravel-datafaker": "^2.2.0",
         "barryvdh/laravel-debugbar": "^3.8",
         "fakerphp/faker": "^1.9.1",
-        "laravel/pint": "^1.8",
+        "laravel/pint": "^1.17",
```

Comment:
Could you please clarify why it's necessary to downgrade the Pint version in this case?

Reply:
@amit-webkul